File: quests/qeynos/Indaria.pl
#Indaria.pl
#Indaria's Doll

sub EVENT_SAY {
	if($text=~/Hail/i)   {
		quest::say("Hello, have you seen my doll?");#Text made up
	}
}

sub EVENT_ITEM {	
	if (plugin::check_handin(\%itemcount, 13712 => 1)) {#Doll
		quest::say("Thank you!  Please take this!");#Text made up
		quest::summonitem(1316);#Velvet Choker
		quest::faction(217, 10);#Merchants of Qeynos
		quest::faction(33, -20);#Circle Of Unseen Hands
		quest::faction(9, 10);#Antonius Bayle
		quest::faction(369, 10);#Coalition of Tradefolk III
		quest::faction(135, 10);#Guards of Qeynos
		quest::exp(27440);#1% level 15 xp
	}
	plugin::return_items(\%itemcount);
}

#End of File zone:qeynos ID: 1082 -- Indaria.pl

File: qeynos/Indaria.lua
-- items: 13712, 1316
function event_say(e)
  if(e.message:findi("hail")) then
    e.self:Say("Hello, have you seen my daughter?");
  elseif(e.message:findi("daughter")) then
    e.self:Say("Yes, my daughter. She went to play near the aqueducts with her doll and now she is gone. If only there was some sign of her..");
  end
end

function event_trade(e)
  local item_lib =require("items");
  if(item_lib.check_turn_in(e.trade, {item1 = 13712})) then
    e.self:Emote("tries to keep her composure. She hands you a silver locket with great effort, and then bursts into tears.");
    e.other:SummonItem(1316); -- Item: Velvet Choker
    e.other:Ding();
    e.other:Faction(262,2,0); -- Guards of Qeynos
    e.other:Faction(219,1,0); -- Antonius Bayle
    e.other:Faction(230,-1,0); -- Corrupt Qeynos Guards
    e.other:Faction(223,-1,0); -- CIrcle of Unseen Hands
    e.other:Faction(291,1,0); -- Merchants of Qeynos
    e.other:AddEXP(27440);
  end
  item_lib.return_items(e.self, e.other, e.trade)
end
